src/systemd_ctypes/bustypes.py

```diff
@@ -432,6 +432,7 @@ class BusType(Enum):
     signature = Annotated[str, StringLikeType('g', is_signature)]
     bytestring = Annotated[bytes, BytestringType('ay')]
     variant = Annotated[dict, VariantType('v')]
+    handle = Annotated[int, FixedType('h', ctypes.c_int)]
```

Comment:
We should maybe pause here for a moment: as a library interface this is kinda OK but Cockpit basically provides a JSON-RPC direct injection mechanism here: that means that javascript world could now cause invalid operations on or leaks of fds to occur just by sending a variant with type `"h"`.

This might be a bit "too easy".  Perhaps we should require this to be a Handle instance to make it a bit more explicit and to provide better handling for cleanup on the receiver side...

Reply:
I did this.